pkg/plugins/plugindef/plugindef.cue

```diff
@@ -18,6 +18,10 @@ schemas: [{
 		id: string & strings.MinRunes(1)
 		id: =~"^([0-9a-z]+\\-([0-9a-z]+\\-)?(\(strings.Join([ for t in _types {t}], "|"))))|(alertGroups|alertlist|annolist|barchart|bargauge|candlestick|canvas|dashlist|debug|datagrid|gauge|geomap|gettingstarted|graph|heatmap|histogram|icon|live|logs|news|nodeGraph|piechart|pluginlist|stat|state-timeline|status-history|table|table-old|text|timeseries|trend|traces|welcome|xychart|alertmanager|cloudwatch|dashboard|elasticsearch|grafana|grafana-azure-monitor-datasource|graphite|influxdb|jaeger|loki|mixed|mssql|mysql|opentsdb|postgres|prometheus|stackdriver|tempo|testdata|zipkin|phlare|parca)$"
 
+		// An alias is useful when migrating from one plugin id to another (rebranding etc)
+		// This should be used sparingly, and is currently only supported though a hardcoded checklist
+		alias?: string
```

Comment:
I also think it would make sense to just have this as a string[] just for sure. For example if we decide to merge 2 data sources into one. Or if we just make some mistake/typo at some point when renaming.

Reply:
updated so it is now `aliasIds` 🎉

Reply:
Nit. Why not `aliases`?